script/Ciel/Day 14/14_01_CIEL14_1B.txt

```diff
@@ -1119,7 +1119,7 @@ I can't allow Arcueid to kill me.
 [sha:01be8f4e9cc388384a8c093bf4e494c635600e3f]{
 -- Page 105, Offset 37338.
 -- 貴方はどんな時でも強がりを言っていた。自分の状態を把握した上で、できもしない事を達成する為に、本気で自分の命を使っていた」
-No matter what happened, you always put on a brave front.
+No matter what happens, you always put on a brave front.
```

Comment:
Speaking about the past, `happened` correct. Put being tricksy by being the same either way.

Reply:
Oh what an obvious error on my part.